src/app/ApproveActivities/components/ApproveActivitiesComponent.js

```diff
@@ -18,12 +18,12 @@ const ApproveActivitiesComponent = ({ activities }) => {
   } else {
     tableBodyHtml = activities.map((activity) => {
       const {
-        id, activityDate, points, description, category,
+        id, createdAt, points, description, category,
```

Comment:
@Chris-Maina You should revert this change since we only use `createdAt` when fetching data from apiary.

Reply:
Yes we can. I just noticed apiary did not have activityDate and thought backend had made changes.

Reply:
Backend is still using activityDate, so if this is merged it might cause a bug when visiting the verify activities page. In this case, we should update Apiary to use activityDate which is much easier.